src/config.c

```diff
@@ -522,10 +522,6 @@ void loadServerConfigFromString(char *config) {
                 err = "Invalid master port"; goto loaderr;
             }
             server.repl_state = REPL_STATE_CONNECT;
-        } else if (!strcasecmp(argv[0],"list-max-ziplist-entries") && argc == 2){
```

Comment:
This technically breaks backwards compatibility, it would be easier to just leave them here.